submitted/rainbond-operator/LICENSE

```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
This License file should be removed as it conflicts with license of this project.

Reply:
Has been removed